File: lambda/src/main/java/model/test/SolvableOpenQuestion.java
package model.test;

import java.io.Serializable;

public class SolvableOpenQuestion implements Serializable {

    private String question;
    private int maxScore;
    private String answer;
    private String correctAnswer;

    public SolvableOpenQuestion() {
    }

    public SolvableOpenQuestion(OpenQuestion openQuestion) {
        this.question = openQuestion.getQuestion();
        this.maxScore = openQuestion.getMaxScore();
        this.correctAnswer = openQuestion.getCorrectAnswer();
        this.answer = "";
    }

    public String getQuestion() {
        return question;
    }

    public void setQuestion(String question) {
        this.question = question;
    }


    public int getMaxScore() {
        return maxScore;
    }

    public void setMaxScore(int maxScore) {
        this.maxScore = maxScore;
    }

    public String getAnswer() {
        return answer;
    }

    public void setAnswer(String answer) {
        this.answer = answer;
    }
}

File: web/src/app/model/solvable-open-question.ts
export class SolvableOpenQuestion {
  question: string;
  correctAnswer: string;
  answer: string;
  maxScore: number;
  receivedScore: number;
  
  constructor(question: string, answer: string, maxScore: number, receivedScore: number, correctAnswer: string) {
    this.question = question;
    this.correctAnswer = correctAnswer;
    this.answer = answer;
    this.maxScore = maxScore;
    this.receivedScore = receivedScore;
  }
}
